ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -1707,6 +1726,9 @@ private class JdbcLedgerDao(
       ()
     }
 
+  override val transactions: TransactionWriter[LedgerOffset] =
+    TransactionWriter.apply(dbDispatcher)
```

Comment:
Can we have a test in `JdbcLedgerDaoSpec` to show how this is used?

Reply:
I agree that we need to test this, but the problem is that there's no read API for this yet and I wanted to keep PR size at the absolute minimum. I was thinking of adding tests to the `JdbcLedgerDao` in a follow up PR. Do you have suggestions?

Reply:
Nah, sounds good.